packages/website/src/hooks/useMediaQuery.ts

```diff
@@ -0,0 +1,46 @@
+// Modified from https://github.com/antonioru/beautiful-react-hooks/blob/master/src/useMediaQuery.ts
```

Comment:
Alternately, how about importing https://github.com/streamich/react-use/blob/master/docs/useMedia.md? I'm surprised it's not already in our `yarn.lock` transitively.

I know I've brought it up in a previous PR conversation but the search & manually crawling through PRs is failing me...

Reply:
Sure, swapped.